static/app/utils/replays/hooks/useCrumbHandlers.tsx

```diff
@@ -13,7 +14,7 @@ function useCrumbHandlers(startTimestampMs: number = 0) {
     setCurrentTime,
   } = useReplayContext();
 
-  const handleMouseEnter = useCallback(
+  const mouseEnterHandler = useCallback(
```

Comment:
This name is a bit confusing, I wouldn't know what it does vs `handleMouseEnter`, but this might be moot if we can get rid of the `hoverDebounce` option.